source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,39 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-06-30
+
+### Highlights
+
+### Unlimited E10/20 trials
+ - Users are now prohibited from starting unlimited E10/20 trials.
+
+### Emoji Enhancements with Skin Tone Selection
+ - Added support for emoji standard v13.0. Users will see Reactions and Emoji rendered based on their operating system or device, and have the ability to choose various skin tones using the Mattermost emoji picker.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added English-Australian English language variation.
+ - Added the ability to upload ``.jpeg`` files on Linux.
+ - Before anything is typed, a list of channels you are a member of are now shown and sorted by recency. The number of channels shown here in the default state are capped to maximum of 20. If there is no recent channel activity (in the case of new users), an alphabetical list of channels is shown showing **My Channels** first and other **Public Channels** next, still capped at 20 for the default state.
+ - Polish, German, and Italian languages were downgraded to beta.
```

Comment:
Does this statement convey everything we need to communicate about this? What do users need to know about a given supported language that's downgraded to beta?

Reply:
Added a note and a link to docs.